mackup/applications/mps-youtube.cfg

```diff
@@ -0,0 +1,6 @@
+[application]
+name = MPS Youtube
+
+[configuration_files]
+.config/mps-youtube/config
+.config/mps-youtube/playlist_v2
```

Comment:
I'm sorry for my behaviour yesterday, I had a really bad day and acted like an idiot.
I've fixed the issue now, thanks @vitorgalvao for pointing it out.